ansi/cursor.go

```diff
@@ -188,3 +188,48 @@ const SaveCursorPosition = "\x1b[s"
 //
 // See: https://vt100.net/docs/vt510-rm/SCORC.html
 const RestoreCursorPosition = "\x1b[u"
+
+// SetCursorStyle (DECSCUSR) returns a sequence for changing the cursor style.
+//
+//	CSI Ps SP q
+//
+// Where Ps is the cursor style:
+//
+//	0: Blinking block
```

Comment:
Both 0 and 2 are blinking blocks?

Reply:
Yes, both 0 and 1 are blinking blocks. See https://vt100.net/docs/vt510-rm/DECSCUSR.html